### PR TITLE
Add meta file to dump

### PR DESCRIPTION
**Pull request by pooknull**

### CODE REVIEW

File: pkg/dump/meta.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	if err != nil {
+		return Meta{}, err
+	}
+	if statusCode != fasthttp.StatusOK {
+		return Meta{}, fmt.Errorf("non-ok status: %d", statusCode)
+	}
+	resp := new(pmmUpdatesResponse)
+	if err := json.Unmarshal(body, resp); err != nil {
+		return Meta{}, fmt.Errorf("failed to unmarshal response: %s", err)
+	}
```
Comment (ollevche) [inline pkg/dump/meta.go:34]: move to a separate func - `requestPMMVersion`

this logic must not depend on external things

File: pkg/dump/meta.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/valyala/fasthttp"
+	"time"
+)
+
+var (
+	version string
```
Comment (ollevche) [inline pkg/dump/meta.go:11]: move to main package

split to GitCommit and GitBranch

File: Makefile [hunk 3]
```diff
@@ -15,10 +16,12 @@ ADMIN_MONGO_PASSWORD?=admin
 
 DUMP_FILENAME=dump.tar.gz
 
-all: build re mongo-reg mongo-insert export-all re import-all
+VERSION:=$(shell git rev-parse --short HEAD)
+
+all: build up mongo-reg mongo-insert export-all export-ch export-vm re import-all
```
Comment (ollevche) [inline Makefile:21]: build re mongo-reg mongo-insert export-all re import-all

**Review by ollevche — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (pooknull) [reply Makefile]: Fixed in bdf6299c3cc59f1cba6b88ae2f8e96250cf60203

File: pkg/dump/meta.go (hunk 2 above)
Comment (pooknull) [reply pkg/dump/meta.go]: Resolved in 87e1bb75e51f6bdb1da9fe675ca1c023e5356ad0

File: pkg/dump/meta.go (hunk 1 above)
Comment (pooknull) [reply pkg/dump/meta.go]: Resolved in 87e1bb75e51f6bdb1da9fe675ca1c023e5356ad0

**Pull request merged**